### DISCUSSION TITLE
WHITELIST not working

### DISCUSSION THREAD

**Discussion by omerhodzic123** [itzg/docker-minecraft-server#1731] [Q&A] [answered]
Hi I'm trying to start a server using the following docker compose file:

mc-vanilla:
    image: itzg/minecraft-server
    ports:
      - 25565:25565
    environment:
      EULA: "TRUE"
      MOTD: "Vanilla Minecraft Server"
      DIFFICULTY: normal
      WHITELIST: dotMonkey, harinun, Cryyptor
      ENFORCE_WHITELIST: "true"
      OPS: dotMonkey
      ICON: https://cdn.discordapp.com/redacted
      ENABLE_RCON: "true"
      RCON_PORT: 25575
      RCON_PASSWORD: redacted
      GENERATE_STRUCTURES: "true"
      SPAWN_PROTECTION: 0
      MEMORY: 4G
      TZ: Australia/Sydney
      OVERRIDE_SERVER_PROPERTIES: "true"
      OVERRIDE_WHITELIST: "true"
      ONLINE_MODE: "true"
    tty: true
    stdin_open: true
    restart: unless-stopped
    volumes:
      - /minecraft/vanilla:/data

However when I try joining the server using my dotMonkey account I get a not on whitelist error. Anyone know what's wrong here?

**Comment by itzg** [comment, marked as answer]
Their API is broken right now. I'm going to work on removing usage of it in #1733

**Reply by omerhodzic123** [reply]
Ahh that would make sense, was trying to use a website to find my uuid and that wasn't working either. Probably using the same api. Thanks for the reply!

**Reply by itzg** [reply]
This should be fixed now

https://github.com/itzg/docker-minecraft-server/issues/1733#issuecomment-1245524846
